rpcperms/interceptor.go

```diff
@@ -772,6 +772,8 @@ func (r *InterceptorChain) rpcStateUnaryServerInterceptor() grpc.UnaryServerInte
 	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo,
 		handler grpc.UnaryHandler) (interface{}, error) {
 
+		r.rpcsLog.Debugf("[%v] requested", info.FullMethod)
```

Comment:
Nice! Not sure if this maybe gets too noisy on nodes with Lit, LNDg and other tool running. So was thinking maybe we could create a new logger just for this? Or maybe it's enough to just set `RPCS=info` if things become too noisy, looks like we don't use the `rpcsLog` for debug or trace outputs in that many places.

Reply:
yeah we can always add a new one when it's too noisy🎧